docs/design/disable-add-in-commands.md

```diff
@@ -14,22 +14,22 @@ You can also specify whether the command is enabled or disabled when the Office
 > [!NOTE]
 > This article assumes that you are familiar with the following documentation. Please review it if you haven't worked with Add-in Commands (custom menu items and ribbon buttons) recently.
 >
-> [Basic concepts for Add-in Commands](add-in-commands.md)
+> - [Basic concepts for Add-in Commands](add-in-commands.md)
 
-## Preview status
+## Host and platform support only
```

Comment:
One of our FHL HackTOC changes was switching out "host" for "application" or "Office application" whenever possible, to avoid confusion with other types of hosts. [This is the commit](https://github.com/OfficeDev/office-js-docs-pr/pull/2062), for reference. This instance seems like a good candidate for that switch, too! 
```suggestion
## Office application and platform support only
```